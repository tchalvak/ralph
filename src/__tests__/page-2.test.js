import React from 'react'
import renderer from 'react-test-renderer'
import Page2 from '../pages/page-2'
import { StaticQuery } from 'gatsby'

beforeEach(() => {
    StaticQuery.mockImplementationOnce(({ render }) =>
        render({
            site: {
                siteMetadata: {
                    title: `Test site name`,
                },
            },
        })
    )
})

describe('Page 2', () => {
    it('renders correctly', () => {
        const location = {
            pathname: '/',
        }

        const tree = renderer.create(<Page2 location={location} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
