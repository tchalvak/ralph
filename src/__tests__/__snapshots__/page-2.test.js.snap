// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page 2 renders correctly 1`] = `
<div
  className="page"
  id="wrapper"
>
  <div>
    <h1>
      Hi from the second page
    </h1>
    <p>
      Welcome to page 2
    </p>
    <a
      href="/"
    >
      Go back to the homepage
    </a>
  </div>
</div>
`;
